components/ListView.js

```diff
@@ -132,7 +135,18 @@ class ListView extends React.Component {
     });
 
     if (this.props.onRefresh) {
-      this.props.onRefresh();
+      const callback = this.props.onRefresh();
+      const setIdle = () => {
+        if(this._isMounted) {
```

Comment:
Do we really need this? Is it even possible that `onRefresh` is called before component is mounted?